### PR TITLE
Add 11.13 release notes

### PR DESCRIPTION
**Pull request by tas50**
#### Description

New week. New release

#### Types of changes

- [ ] Functional documentation bug fix (i.e., broken link or some other busted behavior)
- [ ] New functional doc capabilities (i.e., filter search results)
- [ ] New content
- [ ] Revision to existing content
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

#### Checklist

- [ ] I have read the **README** document about contributing to this repo.
- [ ] I have tested my changes locally and there are no issues.
- [ ] All commits are signed.

### CODE REVIEW

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)
+
+---
+
+## 🎉 NEW FEATURES
+
+### Exceptions for vendor advisories
+
+Exceptions can now be set on advisories, not just checks. To set an exception on an advisory navigate to the advisories page on your asset, and select the advisories you wish to snooze or disable.
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:22]:
```suggestion
You can now set exceptions on advisories, not just checks. To set an exception on an advisory, navigate to an asset's Advisories tab and select the advisories to snooze or disable.
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+For more information, [check out the documentation](https://mondoo.com/docs/platform/security/customize/advisories/).
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:24]:
```suggestion
To learn more, read the [Mondoo documentation](https://mondoo.com/docs/platform/security/customize/advisories/).
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+### Set scoring system per-policy
+
+Mondoo policies support several different scoring systems, depending on how you want to weight the results of the policy. While the recommended scoring mechanism for each policy is built into the policy itself, you can now override the scoring mechanism used directly from the Mondoo console.
+
+![Selecting a scoring system for a policy](/img/releases/2024-07-16-mondoo-11.13-is-out/scoring_selector.png)
+
+### Failing integration notification
+
+Tokens may expire and accounts might change, but now you can quickly spot when integrations stop scanning as they should. Space owners will notice a new icon in the integrations menu whenever an issue arises. Plus, any failing integrations are highlighted on the integrations page, so you can identify the problem type at a glance without diving into each page.
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:37]:
```suggestion
Tokens may expire and accounts might change. Now you can quickly spot when integrations stop scanning as they should. Space owners will notice a new icon in the integrations menu whenever an issue arises. Plus, any failing integrations are highlighted on the Integrations page, so you can identify the problem type at a glance without diving into each page.
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+![Integrations showing failures](/img/releases/2024-07-16-mondoo-11.13-is-out/integrations.png)
+
+#### Updated asset tables on individual checks
+
+The table of affected Assets table on each individual check page is now much more informative, showing last update time, additional risk factors, risk score, and asset name. It also supports multi-select for building targeted cases.
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:43]:
```suggestion
The table of affected assets on each individual check page is now much more informative, showing last update time, additional risk factors, risk score, and asset name. It also supports multi-select for building targeted cases.
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+### Create cases directly from policy check pages
+
+Cases are a new feature of Mondoo that allow you to turn security findings into tasks to complete. They integrate with Atlassian Jira to fit into your existing workflow.
+
+![Mondoo Case Management Screen](/img/releases/2024-07-09-mondoo-11.12-is-out/screenshot_cases.png)
+
+When you see a security finding that requires fixing, you create a case for that finding. Mondoo automatically includes the details of the finding, information on the asset(s) that contain the finding, and instructions for fixing it. When you save the case, Mondoo automatically creates an issue containing all the same information in your Jira project.
+
+To create cases from failing checks, simply open a check and click the 'Create Case' button to get started. You can also [check out the documentation](https://mondoo.com/docs/platform/cases/overview/).
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:56]:
```suggestion
To create cases from failing checks, simply open a check and click the 'Create Case' button. To learn more, read the [Mondoo documentation](https://mondoo.com/docs/platform/cases/overview/).
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+## 🧹 IMPROVEMENTS
+
+### Updated CIS Windows 10 / 11 benchmark policies
+
+Secure Windows endpoints with the latest CIS Windows 10 and 11 benchmark policies version 3.0. This major version bump includes a large refactoring of the recommendations for securing Windows hosts including new and updated recommendations, improved descriptions / remediation text, and overall improvements to queries to ensure you always have the best output to work with.
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:62]:
```suggestion
Secure Windows endpoints with the latest CIS Windows 10 and 11 benchmark policies: version 3.0. This major version bump includes a large refactoring of the recommendations for securing Windows hosts, including new and updated recommendations, improved descriptions and remediation text, and overall improvements to queries to ensure you always have the best output to work with.
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+## 🐛 BUG FIXES AND UPDATES
+
+- Immediatly refresh asset check overview statistics when exceptions are set. - https://github.com/mondoohq/console/pull/4640
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:66]:
```suggestion
- Immediately refresh asset check overview statistics when exceptions are set.
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+- Fix incorrect check impact scores displayed on assets. - https://github.com/mondoohq/console/pull/4633
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:67]:
```suggestion
- Fix incorrect check impact scores displayed on assets.
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+- Ensure editors can't create API tokens with higher privileges. - https://github.com/mondoohq/server/issues/7216
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:68]:
```suggestion
- Ensure editors can't create API tokens with Owner privileges.
```
Am I understanding that correctly? I mean, there's only one step higher.

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+- Allow sorting by blast radius in tables. - https://github.com/mondoohq/server/issues/8328
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:69]:
```suggestion
- Allow sorting by blast radius in tables.
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+- Fix policy check pages not showing all affected assets. - https://github.com/mondoohq/server/issues/8626
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:70]:
```suggestion
- Fix policy check pages not showing all affected assets.
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+- Don't include checks with exceptions in check counts. - https://github.com/mondoohq/server/pull/8628
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:71]:
```suggestion
- Don't include checks with exceptions in check counts.
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+- Expand the data included in data exports to include space metadata, base score, and risk score. - https://github.com/mondoohq/server/issues/8649
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:72]:
```suggestion
- Expand the data included in data exports to include space metadata, base score, and risk score.
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 14]
```diff
@@ -0,0 +1,82 @@
...
+- Fix backwards sorting in risk score table columns. - https://github.com/mondoohq/server/issues/8579
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:73]:
```suggestion
- Fix backwards sorting in risk score table columns.
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 15]
```diff
@@ -0,0 +1,82 @@
...
+- Allow sorting of last updated time in affected asset tables. - https://github.com/mondoohq/server/pull/8631
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:74]:
```suggestion
- Allow sorting by last updated time in affected asset tables.
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 16]
```diff
@@ -0,0 +1,82 @@
...
+- Fix incorrect scores on versioned software pages. - https://github.com/mondoohq/server/issues/8632
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:75]:
```suggestion
- Fix incorrect scores on versioned software pages.
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 17]
```diff
@@ -0,0 +1,82 @@
...
+- Don't scan Azure Storage Containers as part of the `--discover all` command line option. - https://github.com/mondoohq/server/issues/8660
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:76]:
```suggestion
- Don't scan Azure Storage containers as part of the `--discover all` command line option.
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 18]
```diff
@@ -0,0 +1,82 @@
...
+- Use fully qualified Kubernetes names to ensure assets are unique.
+- Show `cnspec status` output when the client fails to communicate with the platform.
+- Substantial improvements to the reliability of multiple Linux CIS benchmark queries.
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:79]:
```suggestion
- Substantially improve the reliability of multiple Linux CIS benchmark queries.
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 19]
```diff
@@ -0,0 +1,82 @@
...
+- Fix format changes introduced to the CSV data export that were not backwards compatible. - https://github.com/mondoohq/server/pull/8673
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:80]:
```suggestion
- Fix format changes introduced to the CSV data export that were not backwards compatible.
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 20]
```diff
@@ -0,0 +1,82 @@
...
+- Fix some CI scan results failing to load in the console. - https://github.com/mondoohq/server/pull/8670
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:81]:
```suggestion
- Fix some CI scan results failing to load in the console.
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 21]
```diff
@@ -0,0 +1,82 @@
...
+- Use more consistent names for out of the box data pack queries. - https://github.com/mondoohq/server/pull/8658
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:82]:
```suggestion
- Use more consistent names for out-of-the-box data pack queries.
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 22]
```diff
@@ -0,0 +1,82 @@
...
+
+### Updated CIS Windows 10 / 11 benchmark policies
+
+Secure Windows endpoints with the latest CIS Windows 10 and 11 benchmark policies: version 3.0. This major version bump includes a large refactoring of the recommendations for securing Windows hosts, including new and updated recommendations, improved descriptions and remediation text, and overall improvements to queries to ensure you always have the best output to work with.
+
+## 🐛 BUG FIXES AND UPDATES
+
+- Immediately refresh asset check overview statistics when exceptions are set.
+- Fix incorrect check impact scores displayed on assets.
+- Ensure editors can't create API tokens with Owner privileges.
```
Comment (tas50) [inline releases/2024-07-16-mondoo-11.13-is-out.md:68]:
```suggestion
- Ensure editors can't create API tokens with higher privileges.
```

File: releases/2024-07-16-mondoo-11.13-is-out.md [hunk 23]
```diff
@@ -0,0 +1,82 @@
...
+### Exceptions for vendor advisories
+
+You can now set exceptions on advisories, not just checks. To set an exception on an advisory, navigate to an asset's Advisories tab and select the advisories to snooze or disable.
+
+To learn more, read the [Mondoo documentation](https://mondoo.com/docs/platform/security/customize/advisories/).
+
+![Selecting advisories to create exceptions](/img/releases/2024-07-16-mondoo-11.13-is-out/advisory_exception.png)
+
+
+### Set scoring system per-policy
```
Comment (misterpantz) [inline releases/2024-07-16-mondoo-11.13-is-out.md:29]:
```suggestion
### Choose the security scoring system for each policy in a space
```

**Review by misterpantz — APPROVED** [review]

**Pull request merged**